docs/pages/BlogPosts.json

```diff
@@ -3,6 +3,7 @@
   "digests": [
     {
       "week": "September 02, 2022",
+      "summary": "Handoff kit updates, Figma search improvements, and more mobile docs",
```

Comment:
Can we split these changes out to a separate PR? It's a pretty different change and we probably want to iterate on it a bit